local/client/conn/conn.go

```diff
@@ -19,6 +20,12 @@ type Dialer interface {
 
 type Conn interface {
 	Echo(arg string) (string, error)
+
+	// TODO(dbentley): this feels weird. We shouldn't expose our internal
```

Comment:
Yeah, this why I feel uneasy about putting this here.

Runner is a useful concept without regard to the local server. It should also be used the Worker binary to run, and the Worker isn't involved in the local server protocol at all.